src/main/java/tr/edu/ogu/ceng/controller/StudentController.java

```diff
@@ -31,10 +32,10 @@ public Student getStudent(@PathVariable(name = "id") long id) {
 	public Student addStudent(@RequestBody Student student) {
 		return studentService.addStudent(student);
 	}
-	
+
 	@PutMapping()
-	 public ResponseEntity <Student> updateStudent(@RequestBody Student student){
-	 return ResponseEntity.ok(student);
+	public ResponseEntity<Student> updateStudent(@RequestBody Student student) {
+		return ResponseEntity.ok(student);
```

Comment:
Bu metotneden studentService çağırmıyor?

Reply:
Hocam bu metotu ben yazmadım , git pull yaptığımda geldi.

Reply:
Zaten değişiklik olarak sadece biçimlendirme var yani boşluk eklenmiş buradaki hata için issue açar mısın?